TalkAloud/TalkAloud/ViewControllers/AudioRecordingsTableViewController.swift

```diff
@@ -10,24 +10,52 @@ import UIKit
 
 class AudioRecordingsTableViewController: UITableViewController {
     
+    var isFiltered = false
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        tableView.register(UINib(nibName: "AudioRecordingCell", bundle: nil), forCellReuseIdentifier: "AudioRecordingCell")
+        tableView.register(UINib(nibName: "AudioRecordingCell", bundle: nil), forCellReuseIdentifier: "AudioRecordingCell")        
     }
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        
         AudioManager.sharedInstance.loadAllRecordings()
+        isFiltered = false
+        tableView.reloadData()
+    }
+    
+    @IBAction func tappedLeftButton(_ sender: Any) {
+        let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        let tagFilterViewController = storyboard.instantiateViewController(identifier: "TagTableViewController") as! TagTableViewController
+        let navigationController = UINavigationController(rootViewController: tagFilterViewController)
```

Comment:
You don't necessary need a UINavigationController to present just one view controller.
If you present a view controller and then you would have navigated to another one that would require to go back to the first, then yeah.

Reply:
So, is there something I should do differently here then?